.github/dependabot.yml

```diff
@@ -0,0 +1,14 @@
+# Dependabot for cargo
+
+version: 2
+updates:
+  # Enable version updates for cargo
+  - package-ecosystem: "cargo"
+    # Look for 'Cargo' files in the `root` directory
+    directory: "/"
+    # Check for updates daily
+    schedule:
+      interval: "daily"
+    reviewers:
+      - "willemml"
+      - "MixedMatched"
```

Comment:
This needs an edit as currently @MixedMatched is responsible for nw-rs/rcas, not nw-rs/rustworks (though that may change in the future).